src/menu.rs

```diff
@@ -6,14 +6,57 @@ use std::{
   hash::{Hash, Hasher},
 };
 
+use crate::platform_impl::{MenuBuilder as PlatformMenuBuilder, MenuItem};
+
+#[derive(Debug, Clone)]
+pub struct MenuBuilder(pub(crate) PlatformMenuBuilder);
```

Comment:
Looks like `Menu` isn't used anymore. Could you rename `MenuBuilder` to `Menu` to replace the previous one?